src/mmw/apps/modeling/serializers.py

```diff
@@ -86,18 +86,20 @@ def validate_weather_type(self, value):
             raise ValidationError('Cannot use Custom Weather Data: '
                                   'none specified.')
 
-        if value == WeatherType.SIMULATION and \
-                not self.instance.weather_simulation:
-            raise ValidationError('Cannot use Simulation Weather Data: '
-                                  'none specified.')
```

Comment:
This validation check cannot be here because the user may set `weather_type = SIMULATION` and provide a value for `weather_simulation` in the same request. That would require payload-level validation rather than field-level validation. I have not looked in to how to do that.